app/middleware/renderer.js

```diff
@@ -1,8 +1,8 @@
 const log = require('../lib/logger');
 const mapLink = require('../lib/mapLink');
 
-function notFound(req, res, msg) {
-  log.warn(msg, 'Not found (404)');
+function notFound(req, res) {
+  log.warn({ req }, 'Not found (404)');
```

Comment:
Yeah, that is pretty much the logic I used to do that here. I was going to look for the request URL but figured, this shouldn't be happening too much and when it does, there may well be some more useful information available in the full request.